Lib/fontTools/ttLib/tables/_g_l_y_f.py

```diff
@@ -408,10 +411,15 @@ def toXML(self, writer, ttFont):
 				writer.begintag("contour")
 				writer.newline()
 				for j in range(last, self.endPtsOfContours[i] + 1):
-					writer.simpletag("pt", [
+					attrs = [
 							("x", self.coordinates[j][0]),
 							("y", self.coordinates[j][1]),
-							("on", self.flags[j] & flagOnCurve)])
+							("on", self.flags[j] & flagOnCurve),
+						]
+					if self.flags[j] & flagReserved1:
+						# Apple's AAT uses flagReserved1 in the first contour/first pt to flag glyphs that contain overlapping contours
```

Comment:
That is an Apple rasterizer feature rather than an AAT-specific one (i.e. not related to layout tables being used).